### PR TITLE
added comming soon and feedback changes

### PR DESCRIPTION
**Pull request by srikanthgudipudi**

### CODE REVIEW

File: src/app/modules/library/learning-resources/learning-resources.component.scss [hunk 1]
```diff
@@ -27,5 +27,13 @@
   height: 3em;
 }
 
+.view-more {
+  margin-top: 5%;
+  padding-left: 10%;
```
[the remark this answers is not in the document]
Comment (srikanthgudipudi) [reply src/app/modules/library/learning-resources/learning-resources.component.scss:32]: No need , removed

File: src/app/modules/library/library.config.ts [hunk 2]
```diff
@@ -1,4 +1,4 @@
 export const LibraryConfig = {
     filterList: 'v1/learningResources/filtersList',
-    learningResources: 'v1/learningResources/all'
+    learningResources: 'v1/learningResources/all?limit=3&page=1'
```
[the remark this answers is not in the document]
Comment (srikanthgudipudi) [reply src/app/modules/library/library.config.ts:3]: changed

File: src/app/modules/portal-shared/card/card.component.html [hunk 3]
```diff
@@ -8,13 +8,15 @@
       </mat-card-subtitle>
     </mat-card-header>
     <mat-card-content>
+      <a href="{{cardData?.url}}" target="_blank">
```
[the remark this answers is not in the document]
Comment (srikanthgudipudi) [reply src/app/modules/portal-shared/card/card.component.html:11]: changed

File: src/app/keycloak-init.ts [hunk 4]
```diff
@@ -0,0 +1,31 @@
+import {environment} from '../environments/environment';
+import {KeycloakService} from 'keycloak-angular';
+
+export function initializer(keycloak: KeycloakService): () => Promise<any> {
+ return (): Promise<any> => {
+   return new Promise(async (resolve, reject) => {
+     try {
+       await keycloak.init({
+         config: {
+            url: 'https://dev.bodh.shikshalokam.org' + '/auth',
```
Comment (kiranharidas187) [inline src/app/keycloak-init.ts:10]: Url should come from env file

File: src/app/keycloak-init.ts [hunk 5]
```diff
@@ -0,0 +1,31 @@
...
+            realm: 'sunbird',
+            clientId: 'sl-ionic-connect',
```
Comment (kiranharidas187) [inline src/app/keycloak-init.ts:12]: These data should be in env. Please don't push private  data

**Pull request merged**